src/player.js

```diff
@@ -1031,6 +1032,9 @@ export default class Player extends FakeEventTarget {
       if (typeof this._config.playback.playsinline === 'boolean') {
         this.playsinline = this._config.playback.playsinline;
       }
+      if (typeof this._config.playback.textStyle === "object") {
+        this.textStyle = Utils.Object.mergeDeep(new TextStyle(), this._config.playback.textStyle);
```

Comment:
Player should know only TextStyle, if someone serializes class data outside of player he should recreate the TextStyle there and set it as player.textStyle = textStyle

Reply:
removed.